nvflare/private/event.py

```diff
@@ -48,7 +48,8 @@ def fire_event(event: str, handlers: list, ctx: FLContext):
 
     if handlers:
         for h in handlers:
-            assert isinstance(h, FLComponent)
+            if not isinstance(h, FLComponent):
+                raise TypeError("h must be FLComponent but got {}".format(type(h)))
```

Comment:
It might be hard for users to understand what is "h"

Reply:
I agree, fixed